product_pricelist_fixed_price/__openerp__.py

```diff
@@ -1,42 +1,20 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    OpenERP, Open Source Management Solution
-#    Copyright (c) 2014 Serv. Tecnol. Avanzados (http://www.serviciosbaeza.com)
-#                       Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as published
-#    by the Free Software Foundation, either version 3 of the License, or
-#    (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
-
+# © 2014 Serv. Tecnol. Avanzados (http://www.serviciosbaeza.com).
+#        Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     "name": "Fixed price in pricelists",
-    "version": "8.0.1.0",
-    "author":  "Serv. Tecnol. Avanzados - Pedro M. Baeza,"
-               "Therp B.V.,"
-               "Odoo Community Association (OCA),"
-               "Giovanni Francesco Capalbo",
+    "version": "8.0.1.0.1",
+    "author": "Serv. Tecnol. Avanzados - Pedro M. Baeza,"
+              "Odoo Community Association (OCA)",
     "category": "Sales Management",
     "website": "www.serviciosbaeza.com",
     "license": "AGPL-3",
     "depends": [
         "product",
     ],
-    "demo": [],
     "data": [
         'view/product_pricelist_item_view.xml',
     ],
-    "init_hook": "init_hook",
```

Comment:
I decided to re-use the method (overriding _auto_end) that was also used in the 7.0 branch. Also because it contained a guard against unneeded or even unwanted changes: only changing base_ext if not already set.